packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,131 @@ related: ['Button']
-#### Compact
-<img src="./img/compact.png" alt="Compact" /> 
+* **Backdrop**
+* **Modal box**
+* **Headline**
+* **Close icon**
+* **Content area**
+* **Buttons** 
 
-The compact modal dialog uses a spacing of 24px. Use this for smaller modals that don’t contain much content.
+See the "[Content](#content)" section for information about writing modal copy.
 
-#### Large
-<img src="./img/large.png" alt="Large" />
+## Spacing
 
-The large modal dialog uses a spacing of 32px. Use this for larger modals that contain a lot of content (code previews, forms, tables...).
+### Compact (24px)
+
+<img src="./img/modal-compact.png" alt="A compact modal with 24px spacers" width="600" />
+
+
+Compact modals have 24px spacing and are best used for modals that contain small amounts of content. 
+
+### Large (32 px)
+
+<img src="./img/modal-large.png" alt="A large modal with 32px spacers" width="675" />
+
+Large modals have a 32px spacing to accommodate a larger content area for elements like code previews, forms, and tables.
 
 ## Usage
-### When to use
-Use a modal dialog when information is critical and requires user input or attention.
-Use modals to:
 
-* [Validate user decisions](#validate-user-decisions)
-* [Report errors](#report-errors)
-* [Provide the result of a task or state of an application](#provide-the-result-of-a-task-or-state-of-an-application)
-* [Get required user input](#get-required-user-input)
+Use a modal to communicate critical information that requires user input or action. 
+
+Modals can:
+
+* Validate user decisions.
+* Confirm non-destructive or destructive actions.
+* Report errors.
+* Share task results and statuses.
+* Prompt required user input.
+
+Don’t use a modal unless its information is vital to user success and worth disrupting a user’s task flow.
+
+## Types of modals
+
+Modals serve several functions in a UI and appear in three main types:
+
+* [Confirmation dialogs](#confirmation-dialogs)
+
+* [Error dialogs](#error-dialogs)
+
+* [Passive dialogs](#passive-dialogs)
+
+### Confirmation dialogs
+
+Use a confirmation dialog to validate user decisions and communicate their consequences. Confirmation dialogs can confirm non-destructive and destructive actions.
+
+#### Confirm a non-destructive action
+
+<img src="./img/non-destructive-confirmation-dialog.png" alt="A side-by-side comparison of unsuccessful and successful confirmaton dialogs for leaving a page without saving. The successful dialog explains the consequence of this action: All unsaved informaton will be lost." />
+
+When confirming a non-destructive action, provide specific context about what will happen once a user confirms it.
+
+#### Confirm a destructive action
+
+<img src="./img/destructive-confirmation-dialog.png" alt="A side-by-side comparison of unsuccessful and successful destructive confirmaton dialogs for deleting a record. The successful dialog explains the consequence of deleting this record: Record name, location, and notes will be lost." />
```

Comment:
Button placement is off! Should be left aligned with the text

Reply:
Also the alert icon seems a little higher than the header! Make sure you're using the modal symbol that has the icon in it in the header